functions-and-arrays.js

```diff
@@ -0,0 +1,208 @@
+// Find the maximum
+function maxOfTwoNumbers(x, y) {
+    if (x > y) {
+        return x;
+    }
+    else {
+        return y;
+    }
+}
+
+// Finding Longest Word
+var words = [
+    'mystery',
+    'brother',
+    'aviator',
+    'crocodile',
+    'pearl',
+    'orchard',
+    'crackpot'
+];
+
+
+function findLongestWord(words) {
+    if (words.length < 1) {
+        return;
+    }
+    var largestWordNum = 0;
+    var largestWord = "";
+    words.forEach(function (word) {
+        if (word.length > largestWordNum) {
+            largestWordNum = word.length;
+            largestWord = word;
+        }
+    });
+    return largestWord;
+}
+
+
+// Calculating a Sum
+
+var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
+
+function sumArray(numbers) {
+    var sum = 0;
+    numbers.forEach(function (num) {
+        sum += num;
+    });
+    return sum;
+}
+
+
+// Calculate the Average
+
+var numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
+
+function averageNumbers(numbersAvg) {
+    if (numbersAvg.length < 1) {
+        return;
+    }
+
+    var toBeAvg = 0;
+    numbersAvg.forEach(function (num) {

```

Comment:
You could've reuse sumArray() function that you created in the iteration before.